ocs_ci/ocs/utils.py

```diff
@@ -690,3 +687,28 @@ def collect_ocs_logs(dir_name):
             f"Timeout {must_gather_timeout}s for must-gather reached, command"
             f" exited with error: {ex}"
         )
+
+
+def collect_ocs_logs(dir_name):
+    """
+    Collects OCS logs
+
+    Args:
+        dir_name (str): directory name to store OCS logs. Logs will be stored
+            in dir_name suffix with _ocs_logs.
+
+    """
+    log_dir_path = os.path.join(
+        os.path.expanduser(ocsci_config.RUN['log_dir']),
+        f"failed_testcase_ocs_logs_{ocsci_config.RUN['run_id']}",
+        f"{dir_name}_ocs_logs"
+    )
+
+    run_must_gather(os.path.join(log_dir_path, 'ocs_must_gather'),
```

Comment:
I see we are running twice below,  what is this call doing?

Reply:
The default image collects everything except worker node service logs (kubelet and crio). We need to run it again to collect those since that is where OCS is actually running.